plasmapy/formulary/dimensionless.py

```diff
@@ -31,10 +31,18 @@
     n_e={"can_be_negative": False},
 )
 def quantum_theta(T: u.K, n_e: u.m ** -3) -> u.dimensionless_unscaled:
-    """
+    r"""
     Compares Fermi energy to thermal kinetic energy to check if quantum
     effects are important.
 
+    The quantum theta (:math:`\theta`) of a plasma is defined by
+
+    .. math::
+        \theta = \frac{E_{T}}{E_{F}}
+
+    where :math:`E_{T}` is the thermal energy of the plasma
+    and :math:`E_{F}` is the Fermi energy of the plasma.
```

Comment:
It would probably make sense to add some notes here on how those are defined (e.g. through a See Also section if we implement them elsewhere, or a quick oneliner for the thermal pressure in the other case). The same applies to `beta`.